packages/trader/src/Stores/client-store.js

```diff
@@ -501,6 +501,11 @@ export default class ClientStore extends BaseStore {
         });
     }
 
+    @computed
+    get residence() {
+        return this.accounts[this.loginid].residence;
```

Comment:
why not `this.residence` here as well? how come each account has its own residence? doesn't client just have one residence?

Reply:
This was brought in, in multiple places, I have created this computed property to be able to `this.residence`. can't use `this.residence` here, will create an infinite loop.

Reply:
I see. but can we store residence once? instead of per each account?